entries/g/goteleport.com.json

```diff
@@ -0,0 +1,15 @@
+{
+    "Teleport": {
+      "domain": "goteleport.com",
+      "tfa": [
+        "totp",
+        "u2f"
+      ],
+      "documentation": "https://goteleport.com/docs/access-controls/guides/webauthn/",
+      "recovery": "https://github.com/gravitational/teleport/discussions/12379",
```

Comment:
The `recovery` entry should include the documentation for users that cannot log in anymore - but these are instructions to generate recovery codes beforehand.
```suggestion
```